### PR TITLE
cryptomator: 1.6.14 -> 1.8.0

### PR DESCRIPTION
**Pull request by evil5519**
###### Description of changes

https://github.com/cryptomator/cryptomator/releases/tag/1.8.0

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by discite** [comment]
This was closed due to a wrong Pull request where the

File: pkgs/tools/security/cryptomator/default.nix [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
   pname = "cryptomator";
-  version = "1.6.14";
+  version = "1.8.0";
 
   src = fetchFromGitHub {
     owner = "cryptomator";
     repo = "cryptomator";
     rev = version;
-    sha256 = "sha256-ArOYL3xj2HiXXu1Bymd5mciMsmikCDvxr5M3LMqZgYA=";
+    sha256 = "sha256-2nCaSL7OlS9f+PZPh0YiMvnjOaAqlQimkKWDSjSP+bQ=";
```
Comment (wegank) [inline pkgs/tools/security/cryptomator/default.nix:15]: Still wrong according to ofborg.

Comment (wegank) [reply pkgs/tools/security/cryptomator/default.nix]: And ofborg now reports this hash. This is so strange!

Comment (discite) [reply pkgs/tools/security/cryptomator/default.nix]: Agree, can we request orgborg to review one more time just in case it changes?

Comment (bachp) [reply pkgs/tools/security/cryptomator/default.nix]: I get `sha256-2nCaSL7OlS9f+PZPh0YiMvnjOaAqlQimkKWDSjSP+bQ=` as a hash when running on x86-64-linux.

Comment (discite) [reply pkgs/tools/security/cryptomator/default.nix]: since is one of my first  time contributing and correcting, what is the best aproach to handle this?

Comment (wegank) [reply pkgs/tools/security/cryptomator/default.nix]: Let's drop the last commit then...

Comment (bachp) [reply pkgs/tools/security/cryptomator/default.nix]: @discite I suggest to [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) and drop all but the first commit.

Comment (bachp) [reply pkgs/tools/security/cryptomator/default.nix]: @discite You most probably also need to change the hash of the dependencies. See line 40.

Comment (SuperSandro2000) [reply pkgs/tools/security/cryptomator/default.nix]: #230757

**Comment by bachp** [comment]
@discite I think there are more changes needed to get cryptomator 1.7+ working. See https://github.com/NixOS/nixpkgs/pull/200590

**Comment by wegank** [comment]
Superseded by #230757.

**Pull request closed without merging**